### ISSUE TITLE
23: Charge parameter timeout reached

### ISSUE THREAD

**Issue by receptanilcengiz** [Sevenstax/FreeV2G#70]
Platform: EVSE
Firmware Version: i.e. V01_00_01
Host Controller Interface: Ethernet
Host: FreeV2G [plug_and_charge](https://github.com/Sevenstax/FreeV2G/tree/plug_and_charge)

Hello

I am charging SERES3 EV. after 15 min I get 23: Charge parameter timeout reached session error

is there any way to increase timeout? or any other solution

**Comment by jpo-stx** [comment]
Hi @receptanilcengiz ,
currently there is no possibility to change this timeout value.
1. You can try to increase the priority of your application:
`$ sudo nice -n -20 python3 Application.py "eth0" -m c4:93:00:22:22:22 -r EVSE`

2. Do you have any other tasks running on your host?
3. What system are using?

BR
Jan

**Comment by receptanilcengiz** [comment]
Hi Jan 

I am running on windows. I have a dotnet app to run python script on cmd.

**Comment by jpo-stx** [comment]
Hi @receptanilcengiz ,

have you tried increasing the prority?

**Comment by receptanilcengiz** [comment]
yes I made it realtime priority which is top priority in windows. but nothing changed.

I think this is not due to priority. I realized that, if I let vehicle to take more current than target, I got this error. (I am not sure). If I give constant current value (ie. 40A) I dont get error. if I send target current value (ie. 200A) to vehicle, I get "23: Charge parameter timeout reached" error. 

I think that communication delay causes this. last condition:
I start charginig -> Vehicle increases target current up to 210A, -> I increases the current to target. -> some times later -> vehicle starts to decrease target current . -> due to delay I decrease current late. and I got error. 

is this possible? 

Vehicle: Tesla Model 3 
SOC: %92
